src/Settings.php

```diff
@@ -680,4 +703,36 @@ public function remove_notices() { // phpcs:ignore Generic.Metrics.NestingLevel.
 			}
 		}
 	}
+
+	/**
+	 * Check the sample file.
+	 *
+	 * @since 1.4.0
```

Comment:
I don't remember whether I shouldhardcode the version number or use {VERSION}, how can we check?

Reply:
@kkarpieszuk npm run gulp build